pkgs/development/compilers/openspin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "openspin-${version}";
+  version = "2018-02-26";
```

Comment:
If you use a commit ref, the version should be in `unstable-yyyy-mm-dd` format.

Reply:
Gotcha - I already had that in the name field and was asked to remove it.  I'll re-add it to the version instead.